### PR TITLE
Ai refactor

### PR DESCRIPTION
**Pull request by Erol444**
Also updated index page:
image

image

### CODE REVIEW

File: source/index.rst [hunk 1]
```diff
@@ -593,8 +497,8 @@ Ecosystem
    :caption: Modules:
 
    C++/Python API <https://docs.luxonis.com/projects/api/en/latest/>
-   Hardware Products <https://docs.luxonis.com/projects/hardware/en/latest/>
    DepthAI SDK <https://docs.luxonis.com/projects/sdk/en/latest/>
+   Hardware Products <https://docs.luxonis.com/projects/hardware/en/latest/>
```
Comment (cafemoloko) [inline source/index.rst:501]:
```suggestion
   Hardware Documentation <https://docs.luxonis.com/projects/hardware/en/latest/>
```

File: source/index.rst [hunk 2]
```diff
@@ -1,17 +1,10 @@
-.. Luxonis Docs documentation master file, created by
-   sphinx-quickstart on Tue Nov  3 14:34:56 2020.
-   You can adapt this file completely to your liking, but it should at least
-   contain the root `toctree` directive.
-
 DepthAI's documentation
```
Comment (cafemoloko) [inline source/index.rst:1]:
```suggestion
DepthAI's Documentation
```

File: source/index.rst [hunk 3]
```diff
@@ -27,7 +20,7 @@ of DepthAI capabilities, which can both encourage you to develop your own ideas
 to discover them yourself.
 
 Demo script
```
Comment (cafemoloko) [inline source/index.rst:22]:
```suggestion
Demo Script
```

File: source/pages/ai_ml_nn.rst [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+`OAK cameras <https://docs.luxonis.com/projects/hardware/en/latest/>`__ can **run any AI model**, even custom
+architectured/built ones. You can even run multiple AI models at the same time, either in parallel or series (a `demo here <https://github.com/luxonis/depthai-experiments/tree/master/gen2-gaze-estimation#gen2-gaze-estimation>`__).
+
+To run a custom AI model on the device, you need to **convert it** to the .blob format - :ref:`documentation here <Converting model to MyriadX blob>`.
+
+You can also choose to use one of **250+ pretrained AI models** from either `OpenVINO Model Zoo <https://github.com/openvinotoolkit/open_model_zoo>`__
+or `DepthAI Model Zoo <https://github.com/luxonis/depthai-model-zoo>`__, read more at :ref:`Use a Pre-trained OpenVINO model`.
+
+Model performance
```
Comment (cafemoloko) [inline source/pages/ai_ml_nn.rst:19]:
```suggestion
Model Performance
```

File: source/pages/troubleshooting.rst [hunk 5]
```diff
@@ -90,33 +90,32 @@ So if you have see this problem with your host, potentially 3 options:
...
-For gen2, set the **usb2Mode** to **True** when creating the device:
-
-.. code-block:: python
-
-  dai.Device(pipeline, usb2Mode=True)
-
-We've also seen an unconfirmed issue of running Ubuntu-compiled libraries on Linux Mint.  If running on not
-Ubuntu 18.04/16.04 or Raspbian, please `compile DepthAI from source <https://docs.luxonis.com/projects/api/en/latest/install/#install-from-source>`__.
-
 Output from DepthAI keeps freezing
```
Comment (cafemoloko) [inline source/pages/troubleshooting.rst:110]:
```suggestion
Output from DepthAI Keeps Freezing
```

File: source/pages/tutorials/first_steps.rst [hunk 6]
```diff
@@ -15,7 +15,10 @@ Let's start with the device setup below
 Connect the DepthAI device
```
Comment (cafemoloko) [inline source/pages/tutorials/first_steps.rst:15]:
```suggestion
Connect the DepthAI Device
```

File: source/pages/troubleshooting.rst [hunk 7]
```diff
@@ -90,33 +90,32 @@ So if you have see this problem with your host, potentially 3 options:
...
-If the output from the device keeps freezing every few seconds, there may be a problem with the USB3 connection and forcing the device into USB2 mode could resolve this issue - instructions are in the chapter above.
+If the output from the device keeps freezing every few seconds, there may be a problem with the USB3 connection and
+forcing the device into USB2 mode could resolve this issue - instructions are in the chapter above.
 
-When connection speed is USB2 (due to some hosts - Windows in particular - or USB controller/port/cable being USB2) - initialization of USB3-enabled firmware or streaming after a few frames may fail. The workaround here is to force the device to use the USB2-only firmware (mentioned in the chapter above).
+When connection speed is USB2 (due to some hosts - Windows in particular - or USB controller/port/cable being USB2) -
+initialization of USB3-enabled firmware or streaming after a few frames may fail. The workaround here is to force the
+device to use the USB2-only firmware (mentioned in the chapter above).
 
 Failed to boot the device: 1.3-ma2480, err code 3
```
Comment (cafemoloko) [inline source/pages/troubleshooting.rst:120]:
```suggestion
Failed to Boot the Device: 1.3-ma2480, err code 3
```

File: source/pages/tutorials/first_steps.rst [hunk 8]
```diff
@@ -15,7 +15,10 @@ Let's start with the device setup below
...
 ##########################
 
-After unboxing your DepthAI package, you will receive your device together with a USB-C cable (and a power supply if you ordered OAK-D)
+After unboxing your DepthAI package, you will receive your device together with a USB-C cable.
```
Comment (cafemoloko) [inline source/pages/tutorials/first_steps.rst:18]: OAK-D-Lite doesn't come with a USB cable.

File: source/index.rst [hunk 9]
```diff
@@ -51,7 +44,7 @@ If you have issues during the installation, see our `Installation page <https://
 instructions
 
 Example Use Cases
-#################
+-----------------
 
 In this section, you'll find an inspiration what can you build right away with DepthAI
```
Comment (cafemoloko) [inline source/index.rst:49]:
```suggestion
In this section, you'll find an inspiration what can you build right away with DepthAI.
```

**Review by cafemoloko — CHANGES_REQUESTED** [review]

**Review by themarpe — APPROVED** [review]

**Pull request merged**